stargz/config/config.go

```diff
@@ -29,6 +29,7 @@ type Config struct {
 	PrefetchSize        int64  `toml:"prefetch_size"`
 	PrefetchTimeoutSec  int64  `toml:"prefetch_timeout_sec"`
 	NoPrefetch          bool   `toml:"noprefetch"`
+	NoLazyPull          bool   `toml:"nolazypull"`
```

Comment:
Because each file content is still lazily pulled on-demand, `nolazypull ` isn't proper term here. `NoBackgroundFetch`  or something should be better